### ISSUE TITLE
A proper logging system

### ISSUE THREAD

**Issue by Libbum** [Libbum/LimnoSES.jl#9]
At present we don't to much logging, and internal statuses are also fairly lax.

There is the `Outcomes` struct but this is in dire straights and needs some better functionality. Monitoring of Pike and Nutrients also is pretty lean. A better system needs to be designed here.

**Comment by SESmodeller** [comment]
Which events need logging that cannot be analyzed post-hoc from the simulated data? If there are none right now, I suggest to close this issue and start a new one instead that focuses on an analytical interface to explore the experiments so far, could that run via Pluto or the shiny-kind of data exploration tool?

**Comment by Libbum** [comment]
Yeah, for the moment the data collector does a good job and really the output is noisy but doesn't need to be anything much better. Having SECC logging was the purpose of this issue though, which isn't reflected in any output analysis methods. It's a low priority regardless.
